test/listen.test.js

```diff
@@ -29,6 +29,17 @@ test('listen accepts a port, address, and callback', t => {
   })
 })
 
+test('listen accepts a port, address, backlog and callback', t => {
+  t.plan(2)
+  const fastify = Fastify()
+  fastify.listen(0, '127.0.0.1', 511, (err) => {
+    fastify.server.unref()
+    t.error(err)
+    t.pass()
```

Comment:
Is there a property that can be checked to make sure the backlog was set correctly?

Reply:
I haven't found a way to get backlog values from server instance. It seems it's just allowing you to override the OS default backlog value when you start a TCP server.